fluent-syntax/src/lib.rs

```diff
@@ -48,4 +48,5 @@
 //! ```
 pub mod ast;
 pub mod parser;
+pub mod serializer;
```

Comment:
Suggestion: Oh one other thing, can you add a description of this change in the CHANGELOG for the crate? This will help me when publishing the change.

Reply:
I added a brief note in the style of other entries there. Do you mean something more verbose when you say description?

Reply:
That'll work, thanks!